packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -387,13 +385,13 @@ class _BottomNavigationTile extends StatelessWidget {
 
     final BottomNavigationBarThemeData bottomTheme = BottomNavigationBarTheme.of(context);
 
-    final double selectedFontSize = selectedLabelStyle.fontSize;
+    final double? selectedFontSize = selectedLabelStyle.fontSize;
```

Comment:
Can you make this non-nullable and just add a `!` on the `selectedLabelStyle.fontSize`? That would eliminate some `!` checks below.  And the checks below get hard to read, because some usages of `selectedFontSize` have them, and some don't need them.

Reply:
Done